llumnix/config/default.py

```diff
@@ -36,6 +36,9 @@
 _C.SERVER.SSL_CERTFILE = None
 # Disable logging requests in server
 _C.SERVER.DISABLE_LOG_REQUESTS_SERVER = False
+# TODO(KuilongCui): Fix default.py cannot take effects to cli arguments of action type.
```

Comment:
Move this to api_server under a parameter with store_true

Reply:
Get